src/track/trackmetadatataglib.cpp

```diff
@@ -838,6 +838,29 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
         parseTrackPeak(pTrackMetadata,
                 toQString(pTrackPeakFrame->fieldList()[1]));
     }
+    TagLib::ID3v2::FrameList ratingFrame = tag.frameListMap()["POPM"];
+    int rating = 0;
+    if(!ratingFrame.isEmpty()) {
+        // RatingString "traktor@native-instruments.de rating=255 counter=2"
+
+        QString sRating = TStringToQString(ratingFrame.front()->toString());
+        sRating = sRating.section("=",1,2).left(3);
+        float fRating = sRating.toInt();
+        rating = ceil(fRating/51);
```

Comment:
I would keep it simple: The Mixxx database should keep track of the Mixxx ratings. 
These can be optionally stored into the file at "mixxx@mixxx.org"

Adopting foreign ratings need an explicit user action like: 
* "Adopt Traktor ratings as Mixxx ratings for unrated tracks"   

This can be simplified with a second read only column that just displays the mean value of any external rating found. 

Or we implement both.